running-coreos/platforms/libvirt/index.md

```diff
@@ -215,3 +163,125 @@ ssh coreos0
 
 Now that you have a machine booted it is time to play around.
 Check out the [CoreOS Quickstart]({{site.baseurl}}/docs/quickstart) guide or dig into [more specific topics]({{site.baseurl}}/docs).
+
+# Running CoreOS cluster demo on libvirt
+
+This guide explains how to run three-nodes demo CoreOS cluster with libvirt.
+
+## Bash Script
+
+Save following bash script into your host filesystem (i.e. `deploy_coreos_cluster.sh`):
+```sh
```

Comment:
Random note: our markdown renderer will need an extra newline before opening this code block